pkg/controller/workspace/workspace_controller.go

```diff
@@ -218,6 +218,12 @@ func (r *ReconcileWorkspace) Reconcile(request reconcile.Request) (reconcileResu
 		return reconcile.Result{}, err
 	}
 
+	if workspace.Annotations[config.WorkspaceStopReasonAnnotation] != "" {
+		workspace.Annotations[config.WorkspaceStopReasonAnnotation] = ""
```

Comment:
I wonder if it would be better if we fully remove stopReason when workspace is started.

Reply:
I'm okay with doing that, @amisevsk wdyt?

Reply:
Yeah I think that makes sense -- if it's not stopped we don't necessarily need a stopped-by annotation :)